guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>In the below lines the code is updated and the use of add_cloudstore() is done but the narratives also need a change accordingly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>